core/src/main/scala/com/mesosphere/usi/core/Scheduler.scala

```diff
@@ -80,14 +97,19 @@ object Scheduler {
     * @param specsSnapshot Snapshot of the current specs
     * @return Snapshot of the current state, as well as Source which produces StateEvents and Sink which accepts SpecEvents
     */
-  def asSourceAndSink(specsSnapshot: SpecsSnapshot, client: MesosClient)(
-      implicit mat: Materializer): (Future[StateSnapshot], Source[StateEvent, NotUsed], Sink[SpecUpdated, NotUsed]) = {
-    val flow = fromClient(client)
+  def asSourceAndSink(
```

Comment:
Why all the re-formats here? Don't we have `scalafmt` to take care of that?

Reply:
sorry, my intellij over-smarted me, reverted the changes.